src/super_gradients/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "3.3.0"
+__version__ = "3.3.1"
```

Comment:
Version change because you want to release a fix already right ?

Reply:
Up!